### DISCUSSION TITLE
How to debug when contributing

### DISCUSSION THREAD

**Discussion by dijdzv** [illixion/vscode-vibrancy-continued#80] [Q&A] [answered]
How can I debug this extension?
I am using windows 11.
I am now running `npm run build-win10` in the project directory, then pressing F5 to check with Extension Host.
At this time, I am having trouble with my code changes not being applied.
I have tried uninstalling Vibrancy Continued, changing settings, etc., but it doesn't seem to help at all.

**Comment by illixion** [comment, marked as answer]
The usual workflow is to clone the repo and open it in VSCode, then open `extension/index.js` and press F5. I haven't tried using npm commands for debugging, so no idea if they still work.

In regards to troubleshooting, you might want to check that you're using a supported theme (for example Dark+) and that your copy of VSCode is up to date. I've ensured that Vibrancy works on Windows 11, so I'm not sure why it wouldn't be working for you.

**Comment by dijdzv** [comment]
I certainly did not need the npm command.
Specifically, I had changed the css, but after changing the code, I was able to apply the changes by running `Reload Vibrancy` and restarting vscode.
Thanks!
